### ISSUE TITLE
pip install failure ReadtheDocs

### ISSUE THREAD

**Issue by bjbford** [ryanfox/sphinx-markdown-tables#9]
requirements.txt:
```
sphinx
sphinx_rtd_theme
recommonmark
sphinx-markdown-tables
```
It seems that this package it not supported by readthedocs.

I believe the issue is from the dependency of python3.5 as it installed fine on my machine, but not the readthedocs server. (The same install error occurred on a colleagues machine with only python3.4) I tried added markdown to the requirements.txt file, however I received the same error.

`pip install -r requirements.txt`:
```bash
Requirement already satisfied: sphinx in /home/docs/checkouts/readthedocs.org/user_builds/casper-toolflow/envs/latest/lib/python2.7/site-packages (from -r docs/requirements.txt (line 1))
Requirement already satisfied: sphinx_rtd_theme in /home/docs/checkouts/readthedocs.org/user_builds/casper-toolflow/envs/latest/lib/python2.7/site-packages (from -r docs/requirements.txt (line 2))
Requirement already satisfied: recommonmark in /home/docs/checkouts/readthedocs.org/user_builds/casper-toolflow/envs/latest/lib/python2.7/site-packages (from -r docs/requirements.txt (line 3))
Collecting sphinx-markdown-tables (from -r docs/requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/d6/b5/eb3e2cb2031bce2593473d6066481e848a382c1368fa1ed9a3832a2fe718/sphinx-markdown-tables-0.0.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-HHUN5_/sphinx-markdown-tables/setup.py", line 5, in <module>
        import sphinx_markdown_tables
      File "sphinx_markdown_tables.py", line 3, in <module>
        import markdown
    ImportError: No module named markdown
    
    ----------------------------------------
```

Maybe this code should be ported to Python 2.7?

**Comment by bjbford** [comment]
Update:
In the ReadtheDocs project Admin page, under Advanced Settings. You can change the Python Interpreter from CPython 2.x (default) to CPython 3.x. This resolved the install error.

I still think revising the code to support Python 2.7 would be a good idea, considering Sphinx works with Python 2.7.

**Comment by ryanfox** [comment]
Seems like there was an issue with a circular import during setup. Only appeared using python 2. At any rate, it should be fixed in v0.0.7.
